[AIX] pip install psutil `v7.1.0` fails due to missing `_psutil_posix.h`

## Summary

- OS: AIX 7.3
- Architecture: PowerPC
- Psutil version: v7.1.0
- Python version: 3.9.23
- Type: installation

## Description

```
      copying psutil/tests/test_linux.py -> build/lib.aix-7302-2345-64-cpython-39/psutil/tests
      copying psutil/tests/test_memleaks.py -> build/lib.aix-7302-2345-64-cpython-39/psutil/tests
      copying psutil/tests/test_misc.py -> build/lib.aix-7302-2345-64-cpython-39/psutil/tests
      copying psutil/tests/test_osx.py -> build/lib.aix-7302-2345-64-cpython-39/psutil/tests
      copying psutil/tests/test_posix.py -> build/lib.aix-7302-2345-64-cpython-39/psutil/tests
      copying psutil/tests/test_process.py -> build/lib.aix-7302-2345-64-cpython-39/psutil/tests
      copying psutil/tests/test_process_all.py -> build/lib.aix-7302-2345-64-cpython-39/psutil/tests
      copying psutil/tests/test_scripts.py -> build/lib.aix-7302-2345-64-cpython-39/psutil/tests
      copying psutil/tests/test_sudo.py -> build/lib.aix-7302-2345-64-cpython-39/psutil/tests
      copying psutil/tests/test_sunos.py -> build/lib.aix-7302-2345-64-cpython-39/psutil/tests
      copying psutil/tests/test_system.py -> build/lib.aix-7302-2345-64-cpython-39/psutil/tests
      copying psutil/tests/test_testutils.py -> build/lib.aix-7302-2345-64-cpython-39/psutil/tests
      copying psutil/tests/test_unicode.py -> build/lib.aix-7302-2345-64-cpython-39/psutil/tests
      copying psutil/tests/test_windows.py -> build/lib.aix-7302-2345-64-cpython-39/psutil/tests
      running build_ext
      building 'psutil._psutil_aix' extension
      creating build/temp.aix-7302-2345-64-cpython-39/psutil
      creating build/temp.aix-7302-2345-64-cpython-39/psutil/arch/aix
      creating build/temp.aix-7302-2345-64-cpython-39/psutil/arch/all
      creating build/temp.aix-7302-2345-64-cpython-39/psutil/arch/posix
      gcc -maix64 -fPIC -pthread -O2 -Wno-unused-result -Wsign-compare -DNDEBUG -g -O2 -Wall -I/opt/freeware/include -I/usr/include -DAIX_GENUINE_CPLUSCPLUS -D_ALL_SOURCE -D_LINUX_SOURCE_COMPAT -I/opt/freeware/include -I/usr/include -DAIX_GENUINE_CPLUSCPLUS -D_ALL_SOURCE -D_LINUX_SOURCE_COMPAT -DPSUTIL_POSIX=1 -DPSUTIL_SIZEOF_PID_T=4 -DPSUTIL_VERSION=710 -DPSUTIL_AIX=1 -I/opt/freeware/include/python3.9 -c psutil/_psutil_aix.c -o build/temp.aix-7302-2345-64-cpython-39/psutil/_psutil_aix.o
      psutil/_psutil_aix.c:54:10: fatal error: _psutil_posix.h: No such file or directory
         54 | #include "_psutil_posix.h"
            |          ^~~~~~~~~~~~~~~~~
      compilation terminated.
      error: command '/opt/freeware/bin/gcc' failed with exit code 1
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for psutil
Successfully built rsmate
Failed to build psutil
error: failed-wheel-build-for-install
Ã Failed to build installable wheels for some pyproject.toml based projects
â°â> psutil
script returned exit code 1
```

--- a/psutil/_psutil_posix.h
+++ b/psutil/_psutil_posix.h
@@ -1,0 +1,9 @@
+/*
+ * Copyright (c) 2009, Giampaolo Rodola'. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+
+long psutil_getpagesize(void);
+int psutil_pid_exists(pid_t pid);
+void psutil_raise_for_pid(pid_t pid, char *msg);

